Builder/BuilderInterface.php

```diff
@@ -22,8 +22,11 @@
 interface BuilderInterface
 {
     /**
-     * @param AdminInterface            $admin
-     * @param FieldDescriptionInterface $fieldDescription
+     * Adds missing information to the given field description from the model manager metadata,
```

Comment:
Ok I think I get it now...
`FormContractor` implements `BuilderInterface` but it's not a real builder like the other builders. It is more like a middle-man between the FormMapper and the "real" Symfony FormBuilder, so the FormMapper needs to call methods on it.
I think this method belongs in fact to the `FormContractorInterface`, because it can be private to the other "normal" builders as no one calls it from the outside.